app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,20 @@
+module SessionsHelper
+# Logs in the given user.
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+  def current_user
```

Comment:
```suggestion
  end
  def current_user
```

các method phân tách nhau bởi lien trống, em self check đi

Reply:
nãy em check rubocop rồi mà quên git add buồn ghê :v